Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,16 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL.
+    ///
+    /// - Parameters:
+    ///   - object: Object of type T that conforms to Encodable protocol.
+    ///   - url: URL to save object to.
+    ///   - encoder: JSONEncoder.
+    /// - Throws: Throws any errors thrown by JSONEncoder or FileManager.
+    public func encode<T: Encodable>(_ object: T, to url: URL, using encoder: JSONEncoder = JSONEncoder(), attributes: [FileAttributeKey: Any]? = nil) throws -> Bool {
```

Comment:
I think you should switch `encoder` and `url` parameters. It reads more naturally because of the `attributes` parameter that it would paired with `url`.
Also, you need to add `attributes` documentation.